gunicorn/sock.py

```diff
@@ -38,6 +38,12 @@ def __getattr__(self, name):
 
     def set_options(self, sock, bound=False):
         sock.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
+        if self.conf.socket_reuseport:
+            try:
+                sock.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEPORT, 1)
+            except AttributeError:
+                # not all platforms have SO_REUSEPORT.
```

Comment:
Would it make more sense to log and/or crash here, instead of silently continuing?